deviceplugin/usb.go

```diff
@@ -18,8 +18,10 @@ import (
 	"crypto/sha1"
 	"encoding/binary"
 	"fmt"
+	"github.com/go-kit/kit/log/level"
```

Comment:
Let's group this separately from the stdlib imports

Reply:
The goland linter continues to be my mortal enemy: 0d70ab4a